src/ProgressBar/ProgressBar.js

```diff
@@ -793,6 +793,8 @@ module.exports = kind(
 		// of the label determination could help here - rjd
 		{path: ['progress', 'popup', '$.popupLabel.content'], method: 'ariaValue'},
 		{path: ['accessibilityValueText'], method: function () {
+			this.setAriaAttribute('aria-valuetext', null);
+			this.setAriaAttribute('aria-valuenow', null);
```

Comment:
We shouldn't null `aria-valuenow` here because that could result in no a11y value if `accessibilityValueText` were changed from a valid value to `null`. Also, there's no need to null `aria-valuetext` just before setting it to another value as it'll incur an extra fn call and DOM access.